scripts/xMetric/addTransactor.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "hardhat";
+import fs from "fs";
+
+async function main() {
+  const newTrasactor = "";
```

Comment:
can this be blank?

Reply:
yeah the intention is that when we run the script against mainnet or whatever we would manually add the person we want to add as a transactor here

Reply:
right but in this context - if there's nothing there will the transaction revert? couldn't you just use an .env variable?

Reply:
oh it'll fail when it tries to convert an empty string into a an `address` -> but yeah -> we aren't adding any transactors right now..  probably not till next week or later in this one?  We can't add a transactor without knowing their address anyways, so this script is just making it easy to plug in their address when we're ready for it

Reply:
ah okay - was slightly confused - If you wanted to - you could just call pause for now and then unpause it when you're ready to do transactions but at least with pause you'll now its working properly

Reply:
I'll add another two scripts to handle pause and unpause but I like having one function per script